wdl/ExpansionHunter.wdl

```diff
@@ -178,19 +181,20 @@ task RunExpansionHunter {
         boot_disk_gb: 10,
         preemptible_tries: 3,
         max_retries: 1,
-        disk_gb: 10 + ceil(size([
-            bam_or_cram,
-            bam_or_cram_index,
-            reference_fasta,
-            reference_fasta_index], "GiB"))
+        disk_gb: 10 + (
+            2 * ceil(size([
+                bam_or_cram,
+                bam_or_cram_index,
+                reference_fasta,
+                reference_fasta_index], "GiB")))
     }
     RuntimeAttr runtime_attr = select_first([runtime_override, runtime_default])
 
     runtime {
         docker: expansion_hunter_docker
         cpu: select_first([runtime_attr.cpu_cores, runtime_default.cpu_cores])
         memory: select_first([runtime_attr.mem_gb, runtime_default.mem_gb]) + " GiB"
-        disks: "local-disk " + select_first([runtime_attr.disk_gb, runtime_default.disk_gb])  + " HDD"
+        disks: "local-disk " + select_first([runtime_attr.disk_gb, runtime_default.disk_gb])  + " SSD"
```

Comment:
Agree with you that it would be good to make SSD vs HDD a part of RuntimeAttr so we can experiment with making this switch in other workflows, but that can be a separate PR.